.circleci/config.yml

```diff
@@ -0,0 +1,77 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/Moya/Moya
+    parallelism: 1
+    shell: /bin/bash --login
+    environment:
+      CIRCLE_ARTIFACTS: /tmp/circleci-artifacts
+      CIRCLE_TEST_REPORTS: /tmp/circleci-test-results
+      LANG: en_US.UTF-8
+      BUNDLE_PATH: vendor/bundle
+    macos:
+      xcode: '9.2.0'
+    steps:
+      - checkout
+      - run: mkdir -p $CIRCLE_ARTIFACTS $CIRCLE_TEST_REPORTS
+      - restore_cache:
+          keys:
+            - v1-dep-{{ .Branch }}-
+            - v1-dep-master-
+            - v1-dep-
+      - run:
+          name: Set Ruby Version
+          command: echo "ruby-2.4" > ~/.ruby-version
+      - run: 
+          name: Fetch CocoaPods Specs
```

Comment:
@freak4pc is this step necessary right now? I think we do `pod install` / `pod update` anymore

Reply:
It might not be necessary :) I didn't dig into it, it was just there in the v1 so I moved it as is. 
Don't mind making a separate PR to get rid of it if that's needed !

Reply:
would be great!